src/components/FactSearchLists/FactSearchList/FactlListItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { ListItem, ListItemText, Typography } from "@mui/material";
+import React from "react";
+import ListItemButton from "@mui/material/ListItemButton";
+import ListItemIcon from "@mui/material/ListItemIcon";
+import { FactlListItemPropTypes } from "@forkfacts/models";
+
+import { useStyles } from "../styles";
+
+export default function FactlListItem({ item, onSelectItem }: FactlListItemPropTypes) {
+  const styles = useStyles();
+
+  return (
+    <ListItem disablePadding onClick={() => onSelectItem(item)}>
+      <ListItemButton>
+        <ListItemIcon>
+          <img src="/recentImg.png" alt={item.name} />
```

Comment:
remove hard coding in `<img src="/recentImg.png">`.  Pass `<img src={item.image}>` instead. Absolutely no hard coding of data in the components.

Reply:
fixed